bin/potion

```diff
@@ -232,6 +235,12 @@ class PotionCommandLine
         end
       end
     end
+
+    def cdq_included?
+      return false unless File.exist?("Gemfile.lock")
+      data = IO.readlines("Gemfile.lock")
+      data.any?{|l| l.match(/\s.cdq\s\(/) }
```

Comment:
could we have a fallback that checks the gemfile as well?

Assuming you've never run bundle, but you are generating models, we'd be making non-CDQ models even though the default template includes CDQ.